src/app/shapes/Triangle.ts

```diff
@@ -1,90 +1,78 @@
-var stage = new Konva.Stage({
-      container: 'container',
+import * as Konva from "konva"; 
+
+let stage = new Konva.Stage({
```

Comment:
Should remove the Konva stage/layer creation from this file--that will happen somewhere else. Triangle simply represents the triangle shape, we will add it to the layer when executing the code